tests/phpunit/tests/block-template.php

```diff
@@ -226,4 +264,109 @@ public function test_resolve_home_block_template_no_resolution() {
 
 		$this->assertNull( $template );
 	}
+
+	/**
+	 * @ticket 55996
```

Comment:
Can we have test descriptions beginning with "Tests" per convention, and covers annotations?

```php
/**
 * Tests that X does/does not Y when Z
 *
 * @ticket 55996
 *
 * @covers
 */
```
Same for other tests in the PR as well.